Editor/Resources/EditorWindow/GameLiftPlugin.uss

```diff
@@ -0,0 +1,59 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+:root {
+    height: 100%;
+    width: 100%;
+    
+    --color-menu-background: #333333;
+    --color-content-background: #3F3F3F;
+    --color-button-primary: #2C5D87;
+    --color-button-hover: #386082;
+    
+    --font-size-menu: 18px;
```

Comment:
I see in follow-up PRs as well as here, we're using a pattern for variables where we have one-to-one mappings of variables to classes. This is one example, but another example is `--font-size-card-button` for the `card__button` class in [PR 94](https://github.com/aws/amazon-gamelift-plugin-unity/pull/94). In my mind the point of variables is not to describe the specific class. Instead it could be something like describing relative options which can be imported from a common `.uss` into each stylesheet so that all the UI has consistent theming.

Example: `--font-size-title` vs `--font-size-subtitle` vs `--font-size-paragraph` imply related use cases, while something simple like `--font-size-small` vs `--font-size-large` imply very basic relations

Reply:
- naming ideas taken on board
- so far as part of this PR, I would saw no
- There is no commonality between the tab buttons and the cta buttons

Reply:
So the main point of this comment was on pulling all these variables into a common uss file that all the other stylesheets will use. I'm fine if we don't do that in this PR but definitely by PR 94 we should start doing this practice

Reply:
I was running on the mindset that GameLiftPlugin.uss was the common file that other pages would inherit, if it could do with a rename I'm open to suggestions. Simply `common.uss`?

Reply:
`GameLiftPlugin.uss` should be specific styling for `GameLiftPlugin.uxml`. Common things like variables and generic class styles would be for a file like `common.uss` or `shared.uss` or `gamelift-theme.uss` something like that

Reply:
I have pulled out styles to be shared into `common.uss`. Currently just variables, but it would include `.button` .`form-row` etc in subsequent PRs